src/Microsoft.Health.Dicom.Core/Features/Query/QueryParser.cs

```diff
@@ -205,21 +211,22 @@ private bool TryParseDicomAttributeId(string attributeId, out DicomTag dicomTag)
             return false;
         }
 
-        private static void ValidateIfTagSupported(DicomTag dicomTag, string attributeId, QueryResource resourceType, IDictionary<DicomTag, ExtendedQueryTagFilterDetails> supportedExtendedQueryTags, out ExtendedQueryTagFilterDetails extendedQueryTagFilterDetails)
+        private static void ValidateIfTagSupported(DicomTag dicomTag, string attributeId, IEnumerable<QueryTag> queryTags, out QueryTag queryTag)
```

Comment:
Since this is currently a `void` method, shouldn't we just return the `out` parameter instead?